app/presenters/document_presenter.rb

```diff
@@ -8,13 +8,31 @@ class DocumentPresenter
     :bulk_published,
     to: :"document.details"
 
+  attr_reader :finder
```

Comment:
Why does this need to be public? I can't see anywhere in this PR where it's called